data/human/human missions.txt

```diff
@@ -4964,3 +4964,77 @@ mission "Riley"
 			label end
 			`	You say your goodbyes, leaving to your ship and returning to the stars.`
 				decline
+
+mission "A dead man's path I"
+	name "Siblings to New China"
+	description "Bring these siblings to New China to hear their father's last will."
+	passengers 2
+	source
+		near "Merak" 3 6
+	destination
+		planet "New China"
+	to offer
+		random < 20
+	on offer
+		conversation
+			`As you walk through the spaceport, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?" You answer, "Indeed, I am." Her face brightens up with a smile, and she continues, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course."`
```

Comment:
```suggestion
			`As you walk through the spaceport, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?"`
			`	You answer, "Indeed, I am."`
			`	Her face brightens up with a smile, and she continues, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course."`
```
it's good to space the text in general and I think it makes sense when smbd else speaks, especially since here you're speaking for the player and choosing they don't lie

Or make this a choice, maybe even a forced one, I think that Amazinite said he didnt like these kind of speak for the player choices

Reply:
Working on it

Reply:
fixed, I think